references/bibtex/hh.bib

```diff
@@ -538506,7 +538506,7 @@ @book{hb:SILPNG:1991-2000
     inlg = {English [eng]},
     macro_area = {Papua}
 }
-@book{e:Lizot:Yanomami,
+@book{e:Lizot:Yanomami:tales,
```

Comment:
@d97hah Don't know how to best disambiguate this from the `e:Lizot:Yanomami` below.